### PR TITLE
Update links to the HTTP specification

### PR DESCRIPTION
**Pull request by xfq**

### CODE REVIEW

**Review by aphillips — APPROVED** [review]
LGTM

File: questions/qa-accept-lang-locales.de.html [hunk 1]
```diff
@@ -133,7 +133,7 @@ <h2>Literaturhinweise</h2>
 
 <ul id="full-links">
     <li>
-      <p><a href="http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html#sec14.4">Was die HTTP-Spezifikation zu Accept-Language sagt</a> </p>
+      <p><a href="https://www.rfc-editor.org/rfc/rfc9110.html#field.accept-language">Was die HTTP-Spezifikation zu Accept-Language sagt</a> </p>
```
Comment (aphillips) [inline questions/qa-accept-lang-locales.de.html:136]: Should we reference STD97 instead?

Comment (xfq) [reply questions/qa-accept-lang-locales.de.html:136]: The problem is that we can't reference the `Accept-Language` request HTTP header of STD 97. We can only reference the section in RFC 9110.

**Comment by xfq** [comment]
These links appear to have been updated. Closing.

**Pull request closed without merging**